modern_pascal_introduction.adoc

```diff
@@ -34,7 +34,7 @@ include::code-samples/hello_world.lpr[]
 This is a complete program that you can _compile_ and _run_.
 
 * If you use the command-line FPC, just create a new file `myprogram.lpr` and execute `fpc myprogram.lpr`.
-* If you use _Lazarus_, create a new project (menu _Project_ -> _New Project_ -> _Simple Program_). Save it as `myprogram` and paste this source code as the main file. Compile using the menu item _Run -> Compile_.
+* If you use _Lazarus_, create a new project (menu _Project_ -> _New Project_ -> _Simple Program_). Save it as `myprogram` and paste this source code as the main file. Compile using the menu item _Run -> Compile_ (F9 hotkey).
 * This is a command-line program, so in either case -- just run the compiled executable from the command-line.
```

Comment:
Note that the hotkey is not F9 on all platforms (in particular, not on macOS). I would prefer to not specify the hotkey here -- users who use the menu item will be able to see the shortcut easily :)

Reply:
Fixed :)